samcli/lib/utils/system_info.py

```diff
@@ -0,0 +1,105 @@
+"""
+Utils for gathering system related info, mainly for use in `sam --info`
+"""
+
+import json
+from typing import Dict, cast
+
+
+def gather_system_info() -> Dict[str, str]:
+    """
+    Gather system info
+
+    Returns
+    -------
+    dict[str, str]
+    """
+    from platform import platform, python_version
+
+    info = {
+        "python": python_version(),
```

Comment:
in the case of MSI and pyinstaller, I'm assuming this gives back the bundled python version information?

Reply:
yes, that should be the python version of the python interpreter running the SAM CLI command.